### PR TITLE
Show announcements on the pre-login screen

### PR DESCRIPTION
**Pull request by parkmuhyeun**
## Description

This PR show announcements on the pre-login screen #918 

## AS-IS

Announcement pops up after logging in

image

## TO-BE

An announcement pops up before you log in. 

image

And the announcement doesn't pop up after logging in.

image

### SecurityConfig.java

Added permission handling for the announcement get method.

### common.css

Since the login screen has different proportions from the index screen, I rearranged it to center it.

### tiles.xml

I removed the unnecessary header layout and injected the scripts layout.

### login.jsp

I removed the boilerplate code and added a meta layout and scripts layout that were already made to be reused.

I also added the code to show the announcement on the login screen.

### template.jsp

Removed related code to prevent announcements from showing up after logging in.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Comments

What fun it was to figure out the original code and fix it 🙌

Any feedback is welcome 🙇🏻‍♂️

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/template/layout/admin/template.jsp [hunk 1]
```diff
@@ -38,7 +28,7 @@
 								<div class="contentsFixBack">
 									<div class="tabMenu">
 										<ul class="nav-tab-menu">
-											<li style="display:none;">
+											<li style=display:none;">
```
Comment (jongwooo) [inline src/main/webapp/WEB-INF/views/template/layout/admin/template.jsp:31]: Please add the missing quotes.

```suggestion
											<li style="display:none;">
```

**Review by jongwooo — CHANGES_REQUESTED** [review]

File: src/main/webapp/WEB-INF/views/template/layout/admin/template.jsp (hunk 1 above)
Comment (parkmuhyeun) [reply src/main/webapp/WEB-INF/views/template/layout/admin/template.jsp]: Thanks, I fixed it 👍🏻

Comment (70825) [reply src/main/webapp/WEB-INF/views/template/layout/admin/template.jsp]: Please combining two commit([6d7bfcc](https://github.com/fosslight/fosslight/pull/924/commits/6d7bfcc04c33af052e87fdde0a3944255cb43eaf), [258b755](https://github.com/fosslight/fosslight/pull/924/commits/258b7556359328ae14c949d8b346195637f53b36)) into one with squash

Comment (parkmuhyeun) [reply src/main/webapp/WEB-INF/views/template/layout/admin/template.jsp]: yep ✓

**Comment by soimkim** [comment]
@parkmuhyeun 님, 정말 정말 죄송합니다.
communication 오류로 인해 해당 기능이 반영되어 본 PR은 Close합니다. 😭

**Pull request closed without merging**